docs/self-managed/setup/deploy/amazon/amazon-eks/eksctl.md

```diff
@@ -291,148 +298,256 @@ For detailed examples, review the [documentation provided by AWS](https://docs.a
+
+The resulting OpenSearch domain is intended for use with the Camunda platform.
+
+### Step-by-Step Setup
+
+1. **Identify the VPC associated with the Amazon EKS cluster:**
+
+   ```shell
+   export VPC_ID=$(aws ec2 describe-vpcs \
+     --query "Vpcs[?Tags[?Key=='alpha.eksctl.io/cluster-name']|[?Value=='$CLUSTER_NAME']].VpcId" \
+     --output text)
+   ```
+
+   The variable `VPC_ID` contains the output value required for the next steps (the value should look like this: `vpc-1234567890`).
+
+2. **Create a security group within the VPC to allow connections to the OpenSearch domain:**
+
+   ```shell
+   export GROUP_ID=$(aws ec2 create-security-group \
+     --group-name opensearch-sg \
+     --description "Security Group to allow internal connections to OpenSearch" \
+     --vpc-id $VPC_ID \
+     --output text)
+   ```
+
+   The variable `GROUP_ID` contains the output (the value should look like this: `sg-1234567890`).
+
+3. **Create a security ingress rule to allow access to OpenSearch over HTTPS (port 443) from within the VPC:**
+
+   ```shell
+   aws ec2 authorize-security-group-ingress \
+     --group-id $GROUP_ID \
+     --protocol tcp \
+     --port 443 \
+     --cidr $CIDR  # Replace with the CIDR range of your EKS cluster, e.g., <EKS_CIDR_BLOCK>
+   ```
+
+   Ensure that the CIDR range is appropriate for your environment.
+
+4. **Retrieve the private subnets of the VPC:**
+
+   ```shell
+   export SUBNET_IDS=$(aws ec2 describe-subnets \
+     --filter Name=vpc-id,Values=$VPC_ID \
+     --query "Subnets[?Tags[?Key=='aws:cloudformation:logical-id']|[?contains(Value, 'Private')]].SubnetId" \
+     --output text | expand -t 1)
+   ```
+
+   The variable `SUBNET_IDS` now contains the output values of the private subnets (the value should look like this: `subnet-0123456789 subnet-1234567890`).
+
+5. **Create the OpenSearch domain:**
+
+   ```shell
+   export OPENSEARCH_DOMAIN_NAME=<your-domain-name>  # Replace with your desired domain name
+
+   aws opensearch create-domain --domain-name $OPENSEARCH_DOMAIN_NAME \
+     --elasticsearch-version 2.15 \
+     --node-to-node-encryption-options Enabled=true \
+     --encryption-at-rest-options Enabled=true \
+     --vpc-options SubnetIds=[${SUBNET_IDS}],SecurityGroupIds=[${GROUP_ID}]
+
+     # TODO: add password and user
+     # TODO: use opensearch, not elastic
+   ```
+
+6. **Wait for the OpenSearch domain to be active:**
+
+   ```shell
+   aws opensearch wait domain-active --domain-name $OPENSEARCH_DOMAIN_NAME
+   ```
+
+7. **Retrieve the endpoint of the OpenSearch domain:**
+
+   ```shell
+   export OPENSEARCH_ENDPOINT=$(aws opensearch describe-domains --domain-names $OPENSEARCH_DOMAIN_NAME --query "DomainStatusList[0].Endpoint" --output text)
+   ```
+
+   This endpoint will be used to connect to your OpenSearch domain.
+
+### Verifying Connectivity from Within the EKS Cluster
+
+To verify that the OpenSearch domain is accessible only from within your Amazon EKS cluster, follow these steps:
```

Comment:
I'd rephrase this as to either say since it's only accessible from within cluster bla bla or remove the only.

Since the following does not check whether it's only accessible from within the cluster.